src/ci/bin/testSingleWorkflowRunner.sh

```diff
@@ -68,3 +68,23 @@ jq '{
 
 cmp <(jq -cS . actual.json) <(jq -cS . expected.json)
 popd > /dev/null
+
+# Test 3: program should exit with error in case if validation of command line arguments failed
+java -jar "${CROMWELL_BUILD_CROMWELL_JAR}" run nonexistent.wdl &
+pid=$!
+sleep 10
+if kill -0 $pid > /dev/null 2>&1; then
+  echo "ERROR: Process still exists"
+  kill $pid
+  exit 1
+fi
+
+# Test 4: application should return non-zero exit code when parsing of command-line arguments fails
+set +e
+java \
+    -jar "${CROMWELL_BUILD_CROMWELL_JAR}" \
+    run ./centaur/src/main/resources/standardTestCases/hello/hello.wdl \
+    ./centaur/src/main/resources/standardTestCases/hello/hello.inputs \
+2>&1 | tee console_output.txt
+set -e
+grep "Error: Unknown argument" console_output.txt
```

Comment:
Should probably also check that `$?` was `1` (per the PR title)

Reply:
Fixed. Lost that part, while adding error message verification.